pkg/monitor/api/drop.go

```diff
@@ -94,6 +94,9 @@ var errors = map[uint8]string{
 	196: "TTL exceeded",
 	197: "No node ID found",
 	198: "Rate limited",
+	199: "IGMP handled",
+	200: "IGMP subscribed",
+	201: "Multicast handled",
```

Comment:
The comment on this struct mentions that these values are shared with `bpf/lib/common.h` and `api/v1/flow/flow.proto`. I see you added these new values to `bpf/lib/common.h`, but I don't see them in `api/v1/flow/flow.proto`. Is there a reason why you didn't add them in there?

Reply:
Not intended! I think I just missed this. Thanks, will fix :)

Reply:
fixed